addons/escoria-ui-keyboard-9verbs/plugin.cfg

```diff
@@ -1,7 +1,7 @@
 [plugin]
 
-name="Escoria 9 Verbs UI"
-description="Classical LucasArts style 9-verbs UI for the Escoria Framework"
-author="StraToN"
+name="Escoria 9 Verbs UI - Gamepad + keyboard"
+description="Classical LucasArts style 9-verbs UI for the Escoria Framework with gamepad support and keyboard control"
```

Comment:
Incidentally, the gamepad support is in simplemouse, not this one.

I added it there because I thought it fit in more naturally with the verb as part of the cursor so you can use the gamepad button to change the verb rather than having to use the joystick to navigate back to the 9verb.

Reply:
@bolinfest oops 😅 i'll edit my commit asap. Sorry